start_bot.py

```
@@ -0,0 +1,112 @@
+    return files_stats
+
+
+def main(title):
+    output_dir = Path(__file__).parent / "svg_data"
+
+    files_stats = start_on_template_title(title, output_dir=output_dir, titles_limit=None, overwrite=False)
+
+    print(f"len files_stats: {len(files_stats):,}")
+
+    site = mwclient.Site('commons.wikimedia.org')
+    site.login(username, password)
```

Ruff (0.14.0)

Undefined name `username`

(F821)